src/main/java/net/wurstclient/hacks/InstaBuildHack.java

```diff
@@ -38,23 +51,31 @@ public final class InstaBuildHack extends Hack
 		"Determines what to build.\n\n"
 			+ "Templates are just JSON files. Feel free to add your own or to edit / delete the default templates.\n\n"
 			+ "If you mess up, simply press the 'Reset to Defaults' button or delete the folder.",
-		"autobuild", path -> {});
+		"autobuild", DefaultAutoBuildTemplates::createFiles);

```

Comment:
Question: Did you run into any problems with InstaBuild's template setting doing nothing here? I was assuming, since AutoBuild's template setting already creates the default templates and both hacks share the same folder, that InstaBuild shouldn't try to create the same templates again.